### PR TITLE
Product list

### PR DESCRIPTION
**Pull request by gopalbbharadva**
**This is the PR for Product listing page of B-shop.**
 
Description:
- I used grid for product listing.
- Click on Explore now CTA button to view products.

### CODE REVIEW

**Review by adidoshi — APPROVED** [review]
The UI looks clean & using grids the code is well structured. Just a small suggestion, if price element could be bolder.

File: Pages/productlist.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
+.main-wrapper {
+  height: 90vh;
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: auto;
```
Comment (msdeshmukh009) [inline Pages/productlist.css]: You can skip this line of code I guess it will anyway make rows implicitly. Correct me if I am wrong.

**Review by msdeshmukh009 — APPROVED** [review]
UI is looking really good!!

File: Pages/productlist.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+.clear-link {
+  cursor: pointer;
+  text-transform: uppercase;
+  font-family: sans-serif;
+  color: var(--Primary_Color);
+}
+.cat-heading {
+  padding: 10px 0;
+  font-family: "Barlow", sans-serif;
+}
```
Comment (himadri2110) [inline Pages/productlist.css]: Suggestion: You can store the `font-family` in a variable and reuse it.

Rest all looks good.

**Review by himadri2110 — APPROVED** [review]

File: Pages/productlist.css (hunk 1 above)
Comment (gopalbbharadva) [reply Pages/productlist.css]: Ya it will make rows implicitly, thanks for telling.

**Pull request merged**